text/profiles/0237-profiles-data-model.md

```diff
@@ -0,0 +1,695 @@
+|BenchmarkAverageArrays-10|7,742|3,311|584|1|869,648|880|
+
+#### Average profile with timestamps added to each sample
+
+The source is the same as in the previous example, but this time there were timestamps added to each sample in the profile. `arrays` representation remains the most efficient one.
+
+|name|bytes|gzipped_bytes|retained_objects|unique_label_sets|bytes_allocated|allocs|
+|---|---|---|---|---|---|---|
+|BenchmarkAverageTimestampsPprof-10|9,478|3,865|n/a|1|n/a|n/a|
+|BenchmarkAverageTimestampsDenormalized-10|119,526|4,154|4,482|1|1,125,320|4,893|
+|BenchmarkAverageTimestampsNormalized-10|9,637|3,549|941|1|890,192|1,873|
+|BenchmarkAverageTimestampsArrays-10|9,560|3,381|684|1|894,472|1,680|
+
+#### Ruby profile with very deep stacktraces
+
+The source for this test is an aggregated pprof profile collected from a Ruby application that has very deep stacktraces. You can see that the `arrays` representation is the most efficient in terms of memory allocations and size of the resulting protobuf payload (``).
+
+|name|bytes|gzipped_bytes|retained_objects|unique_label_sets|bytes_allocated|allocs|
+|---|---|---|---|---|---|---|
+|BenchmarkRuby1Pprof-10|1,869,563|115,323|n/a|1|n/a|n/a|
+|BenchmarkRuby1Denormalized-10|163,107,484|4,716,442|3,840,093|1|319,473,752|3,844,625|
+|BenchmarkRuby1Normalized-10|1,931,905|130,556|41,457|1|18,982,328|78,242|
+|BenchmarkRuby1Arrays-10|1,886,964|120,382|23,481|1|18,365,800|42,284|
+
+#### Large profile
+
+The source for this test is an aggregated pprof profile collected from a Go application over a large period of time. This one shows that the `arrays` representation remains the most efficient, however, the `gzipped_bytes` number is a little bit larger.
+
+|name|bytes|gzipped_bytes|retained_objects|unique_label_sets|bytes_allocated|allocs|
+|---|---|---|---|---|---|---|
+|BenchmarkLargePprof-10|16,767,419|4,625,632|n/a|163|n/a|n/a|
+|BenchmarkLargeDenormalized-10|969,446,655|71,246,645|24,036,740|163|2,140,322,432|24,429,954|
+|BenchmarkLargeNormalized-10|17,813,931|4,874,456|2,068,600|163|208,793,736|4,217,313|
+|BenchmarkLargeArrays-10|16,980,323|5,779,036|964,989|163|218,418,624|3,396,515|
+
+### Semantic Conventions
+
+TODO: describe things like profile types and units
+
+
+### Decision Log
+
+There were many other alternatives considered during the design process. See [Decision Log](https://github.com/open-telemetry/opentelemetry-proto-profile/blob/54bba7a86d839b9d29488de8e22d8c567d283e7b/opentelemetry/proto/profiles/v1/decision-log.md#L0-L1) for more information about various decisions that were made during the design process.
+
+## Open questions
+
+Client implementations are out of scope for this OTEP. At the time of writing this we do have a reference implementation in Go, as well as a working backend and collector, but they are not yet ready for production use. We are also working on a reference implementation in Java. We are looking for contributors to help us with other languages.
```

Comment:
This looks like you have enough prototypes for us to evaluate for actual Specification inclusion.

I'd recommend as soon as we start see approvals landing on this OTEP to open the DataModel specification markdown with just the datamodel components.